docs/features/interceptors.md

```diff
@@ -3,7 +3,7 @@
 ## Summary
 [summary]: #summary
 
-*Interceptors* are an experimental compiler feature planned to ship in .NET 8 (with support for C# only). The feature may be subject to breaking changes or removal in a future release.
+*Interceptors* is a C# compiler feature, first shipped experimentally in .NET 8.
```

Comment:
nit: Consider recording the version where it'll be "shipped"

Reply:
I am expecting the "stable" designation to arrive in 9.0.2xx, so that's what I'll record here.